draft-ietf-quic-load-balancers.md

```diff
@@ -217,7 +222,7 @@ ID is unroutable. These connection IDs MUST self-encode their length (see
 Servers with no active configuration SHOULD provide the client exactly one CID
 over the life of the connection. In QUIC versions 1 and 2, therefore, servers
 SHOULD NOT send any NEW_CONNECTION_ID frames, instead delivering a single CID
-via its first Initial packet.
+via the Source Connection ID of long headers it sends.
```

Comment:
Somewhat unrelated: is this advice we really want to give?   If you only receive one connection ID ever, you can't ever migrate and you can't use short idle periods as an opportunity to break linkability.